src/plugins/data_views/common/data_views/data_view.test.ts

```diff
@@ -287,12 +287,9 @@ describe('IndexPattern', () => {
         '@tags': runtime,
         runtime_field: runtimeField.runtimeField,
       });
-      const field = indexPattern.toSpec()!.fields!['@tags'];
+      const field = indexPattern.fields.toSpec()!['@tags'];
```

Comment:
Sounds good. I had originally made these changes because I had defaulted `includeFields` to `false`, but later realized a lot of consumers were relying on `field` being returned in `toSpec`.